blocks/waves/index.php

```diff
@@ -24,8 +21,7 @@
 		filemtime( plugin_dir_path( __FILE__ ) . 'waves.js' ),
 		true // in footer
 	);
+	wp_set_script_translations( 'block-experiments', 'waves' );
 } );
 
-add_action( 'enqueue_block_editor_assets', function() {
-	wp_enqueue_script( 'a8c-waves-js' );
-} );
+add_filter( 'should_load_separate_core_block_assets', '__return_true' );
```

Comment:
I don't think that an individual block should enable this flag because it's something that should be reserved mostly for themes or site owners. Se the full dev note for more context: https://make.wordpress.org/core/2021/07/01/block-styles-loading-enhancements-in-wordpress-5-8/.